cli/script/command-executor.ts

```diff
@@ -224,6 +238,75 @@ function appRename(command: cli.IAppRenameCommand): Promise<void> {
         });
 }
 
+function appTransfer(command: cli.IAppTransferCommand): Promise<void> {
+    throwForInvalidEmail(command.email);
+
+    return getAppId(command.appName)
+        .then((appId: string): Promise<void> => {
+            throwForInvalidAppId(appId, command.appName);
+
+            return confirm()
+                .then((wasConfirmed: boolean): Promise<void> => {
+                    if (wasConfirmed) {
+                        return sdk.transferApp(appId, command.email)
+                            .then((): void => {
+                                log("Successfully transferred the ownership of \"" + command.appName + "\" to the account with email \"" + command.email + "\".");
```

Comment:
I did have the type "app" in there before, but I got following feedback on it:

`Successfully transferred the ownership of \"" + command.appName + "\" app to the account with email \"" + command.email + "\".`

Comment: "We can probably drop the use of "app" after displaying the app name."

`"Successfully added the \"" + command.email + "\" as a collaborator to \"" + command.appName +"\" app."`

Comment: "Successfully added EMAIL as a collaborator to APP"

To me both of them seem the same, so not really picky about either. Do you still want me to change it?